config/crawler-user-agents.json

```diff
@@ -118,15 +127,43 @@
     "types": [
       "bad"
     ]
-  },
+  }
+  ,
   {
     "pattern": "APIs-Google",
     "addition_date": "2017/08/08",
     "url": "https://support.google.com/webmasters/answer/1061943?hl=en",
     "instances": [
       "APIs-Google (+https://developers.google.com/webmasters/APIs-Google.html)"
     ]
-  },
+  }
+  ,
+  {
+    "pattern": "Google-InspectionTool",
```

Comment:
For the newly added bot entries that we want to filtered out when we use this library, we must add `"types":["bad"]` to them.

We use that type to filter down to just the bot entries we do not want to consider as traffic that should be included in Site Analytics.

Reply:
Newer versions of this library got rid of that and its why we forked.

Reply:
How do we determine which bots are "bad"?

Reply:
Discussed with @housepage.  Each new user agent was investigated to determine if it was something like a crawler for search engines, uptime monitors, etc or if it could be legitimate use of our APIs that we would want to track.